src/selectors/identity.js

```diff
@@ -26,7 +26,7 @@ export const selectClaims = (state) => state.identity.getIn(['personalInformatio
 export const selectClaimCategoriesToDisplay = createSelector(
   [selectClaimCategories, selectShowEmptyClaimCategories, selectClaims],
   (claimCategories, showEmptyClaimCategories, claims) => {
-    if (!showEmptyClaimCategories) {
+    if (showEmptyClaimCategories) {
```

Comment:
This should not be changed, we can only invert the logic for checked prop on the checkbox.